builtin/providers/azurerm/import_arm_virtual_machine_scale_set_test.go

```diff
@@ -31,6 +31,29 @@ func TestAccAzureRMVirtualMachineScaleSet_importBasic(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVirtualMachineScaleSet_importBasic_managedDisk(t *testing.T) {
+	resourceName := "azurerm_virtual_machine_scale_set.test"
+
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMVirtualMachineScaleSet_basic, ri, ri, ri, ri, ri, ri, ri, ri)
```

Comment:
@sozercan so currently this test is identical to the one above it `TestAccAzureRMVirtualMachineScaleSet_importBasic`, as it's running the same test with the same inputs/outputs.

Instead - I believe this test was meant to use the `testAccAzureRMVirtualMachineScaleSet_basicLinux_managedDisk` config? If so - is it possible to switch this out as you've identified? e.g.

```
config := fmt.Sprintf(testAccAzureRMVirtualMachineScaleSet_basicLinux_managedDisk, ri, ri, ri, ri, ri, ri)
```

Thanks :)